### PR TITLE
dts: asus-z010d: Add support for Asus Zenfone Max ZC550KL (Z010D)

### PR DESCRIPTION
**Pull request by devajithvs**

### CODE REVIEW

File: dts/msm8916/msm8916-asus-z010d.dts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "skeleton.dtsi"
+
+/ {
+	qcom,msm-id = <206 0>;
+	qcom,board-id = <0x8 0x0>;
```
Comment (Minecrell) [inline dts/msm8916/msm8916-asus-z010d.dts:9]: This is the same as `msm8916-mtp.dts` so you need to add an entry there and match something from the command line. You can check the command line using `fastboot oem cmdline && fastboot get_staged /dev/stdout`.

Comment (devajithvs) [reply dts/msm8916/msm8916-asus-z010d.dts:9]: This the the output of ```fastboot oem cmdline && fastboot get_staged /dev/stdout```

```
lk2nd console=logk0 androidboot.cpu.serialno=14e4d011  androidboot.asus.warranty=0  CPU_RV=007050e1 CPU_FEATURE_ID=11 loglevel=0 androidboot.cpu.security=enable  pcb_rev=0  androidboot.emmc.type=16G  SB=Y androidboot.rf.sku.id=2 androidboot.serialno=G7AXB7C162483SF androidboot.emmc=true androidboot.mode=charger_shipping androidboot.baseband=msm mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_r69339_720p_video:1:noneOKAY
```

I'm guessing CPU_FEATURE_ID=11 is the one I'm looking for

Comment (Minecrell) [reply dts/msm8916/msm8916-asus-z010d.dts:9]: That one might work, alternatively you can also match `mdss_mdp.panel=` if you want.

Comment (devajithvs) [reply dts/msm8916/msm8916-asus-z010d.dts:9]: Done.

Comment (Minecrell) [reply dts/msm8916/msm8916-asus-z010d.dts:9]: you still need to move it to `msm8916-mtp.dts`

Comment (devajithvs) [reply dts/msm8916/msm8916-asus-z010d.dts]: I think this should work

Comment (devajithvs) [reply dts/msm8916/msm8916-asus-z010d.dts]: Should I squash these commits into one?

File: dts/msm8916/msm8916-mtp.dts [hunk 2]
```diff
@@ -23,4 +23,10 @@
 		// FIXME: There are multiple possible panels, might need to add more
 		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_hx8394a_hd_tdt_video:1:none";
 	};
+
+	asus-z010d {
+		model = "Asus Zenfone Max ZC550KL";
+		compatible = "asus,z010d", "qcom,msm8916", "lk2nd,device";
+		lk2nd,match-cmdline= "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_r69339_720p_video:1:none";
```
Comment (Minecrell) [inline dts/msm8916/msm8916-mtp.dts:30]:
```suggestion
		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_r69339_720p_video:1:none";
```

**Review by Minecrell — CHANGES_REQUESTED** [review]
With this change and if you squash your commits it looks fine to me.

File: dts/msm8916/msm8916-asus-z010d.dts (hunk 1 above)
Comment (Minecrell) [reply dts/msm8916/msm8916-asus-z010d.dts]: > I think this should work

Please test that it works :)

> Should I squash these commits into one?

Yes, please :)

Comment (devajithvs) [reply dts/msm8916/msm8916-asus-z010d.dts]: > Please test that it works :)

Did that already :+1: 

> Yes, please  :)

In a minute

**Review by Minecrell — APPROVED** [review]

**Pull request merged**